hibernate-core/src/main/java/org/hibernate/cfg/annotations/QueryBinder.java

```diff
@@ -67,6 +67,7 @@ public static void bindQuery(
 				.setReadOnly( hints.getBoolean( queryName, QueryHints.READ_ONLY ) )
 				.setComment( hints.getString( queryName, QueryHints.COMMENT ) )
 				.setParameterTypes( null )
+				.setPassDistinctThrough(hints.getPassDistinctThrough(queryName, QueryHints.PASS_DISTINCT_THROUGH))
```

Comment:
Hibernate code format has the convention to use space char delimiters aggressively( two exceptions I know are: 1)method declaration; 2) method invocation with empty parameter). So the above statement should be changed as:
```
.setPassDistinctThrough( hints.getPassDistinctThrough( queryName, QueryHints.PASS_DISTINCT_THROUGH ) )
```

Reply:
Thanks, updated my code style and I think I fixed all the places